pkgs/by-name/sp/spdk/patches/configure.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/configure b/configure
```

Comment:
This patch is now also in latest upstream: https://github.com/spdk/spdk/commit/c3618c42ac3f6fdfcc9c04e29953fd6cf4f71c11